website/content/api-docs/secret/transit.mdx

```diff
@@ -1307,7 +1307,7 @@ using the [`/sys/plugins/reload/backend`][sys-plugin-reload-backend] endpoint.
 
 - `size` `(int: 0)` - Specifies the size in terms of number of entries. A size of
   `0` means unlimited. A _Least Recently Used_ (LRU) caching strategy is used for a
-  non-zero cache size.
+  non-zero cache size. Must be 0 (default) or a value greater or equal to 10 (minimum cache size).
```

Comment:
Maybe you can make it look like the `size` parameter above, including explaining what 0 means?

Reply:
Hmm, I was confused thinking that the new text was for a different parameter, so please disregard my comment.